CMakeLists.txt

```diff
@@ -329,6 +329,8 @@ set(types_files
 )
 
 set(utils_files
+    ${nupic_src}/utils/CSVReader.cpp
```

Comment:
consider using https://github.com/ben-strasser/fast-cpp-csv-parser/blob/master/csv.h 
I've worked with it, it's include only, might be a better fit.